samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -95,11 +96,11 @@ static int remove_whitespace(char *buf)
 static int query_modem(const char *cmd, char *buf, size_t buf_len)
 {
 	int ret;
-	enum at_cmd_state at_state;
 
-	ret = at_cmd_write(cmd, buf, buf_len, &at_state);
+	ret = nrf_modem_at_cmd(buf, buf_len, "%s", cmd);
 	if (ret) {
-		LOG_ERR("at_cmd_write [%s] error:%d, at_state: %d", cmd, ret, at_state);
+		LOG_ERR("nrf_modem_at_cmd[%s] error:%d, at_state: %d", cmd, ret,
```

Comment:
`at_state` doesn't sound like correct for printing given that's a terminology of the old interface.
@lemrey should comment what we should print here given the different variations of the return value.
If `nrf_modem_at_err` is used, also `nrf_modem_at_err_type` should be used.

Reply:
Agree, either remove "at_state" from the print or print both the error type and error value.